app/assets/javascripts/active_admin/base.js

```diff
@@ -506,4 +507,8 @@
     return $("#active_admin_content .tabs").tabs();
   };
   $(document).ready(onDOMReady$2).on("page:load turbolinks:load", onDOMReady$2);
+  exports.ModalDialog = ModalDialog;
```

Comment:
Wouldn't we need an additional export for the camelcase format since others would still be using it?

Reply:
Right, I saw the `N.B.` in the PR description but I think @javierjulio is right. We _could_ export it and warn and suggest the better alternative when it's used so users start changing their code without it being broken.

Reply:
I didn't think about that. That's a great idea. Can we export `modal_dialog` but have it display a console warning (deprecation message) if used?

Reply:
That was my first idea, and that's precisely what I was referring to by "legacy code". Seeing the lack of reactions on the issue and the fact that people having the issue would land on the issue and this PR, I thought it would be unnecessary 🤷‍♂ 
Added back the legacy export with console warning

Reply:
Yeah, I see, sorry for the back and forth :pray:.

I can see how people that already got bit by this issue won't really care about it since the upgrade already broke them. But I think this can help people who haven't yet upgraded and will jump directly to the next version that won't break their applications and show them a warning instead.